hack/helm_vars/wire-server/values.yaml

```diff
@@ -0,0 +1,225 @@
+tags:
+  nginz: true
+  brig: true
+  galley: true
+  gundeck: true
+  cannon: true
+  cargohold: true
+  spar: true
+  proxy: false
+  webapp: false
+  team-settings: false
+  account-pages: false
+  legalhold: false
+
+cassandra-migrations:
+  cassandra:
+    host: cassandra-ephemeral
+    replicaCount: 1
+elasticsearch-index:
+  elasticsearch:
+    host: elasticsearch-ephemeral
+    index: directory_test
+  cassandra:
+    host: cassandra-ephemeral
+
+brig:
```

Comment:
I think we should set `imagePullPolicy` to `Always` for all these images because I imagine we won't be changing the tag very often and kubernetes will refuse to pull the image if it sees it present locally.

Reply:
These values, at the time of this PR, are meant for CI, and CI will bump versions all the time.

I agree this can become an issue when we want to make use of this for local development (i.e. when we compile docker images locally). That is a future PR. Noted in hack/FUTUREWORK.md